### ISSUE TITLE
Possible to spawn outside of map, league_voting

### ISSUE THREAD

**Issue by venomltd** [csgo-league/csgo-league-game#5]
Sometimes you will spawn outside of the map.

**Comment by Pannboo** [comment]
See latest commit. This is now not causing the issue as often but needs to be further tested before closing bug.
EDIT: This commit did not fix the issue. Still monitoring.

**Comment by Pannboo** [comment]
https://github.com/csgo-league/csgo-league-game/commit/a42c68e043471c7a2db42899e09d9e96ae63ce87 This commit adds a 0.5 second timer which should fix this issue but it still needs to be tested before we can close.

**Comment by B3none** [comment]
A hacky solution but if it works it works :man_shrugging::wink:

**Comment by Pannboo** [comment]
Well it's related to how valve handles connections. If we do this on connection there is a risk of the team being assigned at the camera position. I could potentially trial it without the timer and just respawn the player. That may work but for now for safety I have set a 0.5 timer which assigns the team and i respawn them just for safety.

If this test works then I will remove the timer and just respawn and see if it fixes the issue, if that fixes the issue then the timer is no longer needed.
